content/300-guides/400-migrate-to-prisma/02-migrate-from-sequelize.mdx

```diff
@@ -0,0 +1,1217 @@
+
+## Overview of the sample project
+
+For this guide, we'll use a REST API built with Express as a [sample project](https://github.com/prisma/migrate-from-sequelize-to-prisma) to migrate to Prisma. It has four models/entitites:
+
+<TabbedContent tabs={[<FileWithIcon text="User.js" icon="file"/>, <FileWithIcon text="Post.js" icon="file"/>, <FileWithIcon text="Profile.js" icon="file"/>, <FileWithIcon text="Category.js" icon="file"/>]}>     
+ 
+<tab>
+
+```js
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('User', {
+    name: {
+      type: DataTypes.STRING,
+    },
+    email: {
+      type: DataTypes.STRING,
+      unique: true,
+      allowNull: false
+    },
+  });
+
+  User.associate = models => {
+    User.hasMany(models.Post, {
+      foreignKey: 'authorId',
+      as: 'posts'
+    });
+    User.hasOne(models.Profile, {
+      onDelete: 'CASCADE',
+      foreignKey: 'userId'
+    })
+  };
+  return User;
+}
+```
+
+</tab>
+
+<tab>
+
+```js
+module.exports = (sequelize, DataTypes) => {
+  const Post = sequelize.define("Post", {
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    content: {
+      type: DataTypes.STRING,
+    },
+    published: {
+      type: DataTypes.BOOLEAN,
+      defaultValue: false,
+    },
+  })
+  Post.associate = (models) => {
+    Post.belongsTo(models.User, {
+      foreignKey: "authorId",
+      as: "author",
+    })
+    Post.belongsToMany(models.Category, {
+      through: "PostCategories",
+      as: "categories",
+    })
+  }
+  return Post
+}
+```
+
+</tab>
+
+<tab>
+
+```js
+module.exports = (sequelize, DataTypes) => {
+  const Profile = sequelize.define("Profile", {
+    bio: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  })
+  Profile.associate = (models) => {
+    Profile.belongsTo(models.User, {
+      foreignKey: "userId",
+      as: "user",
+    })
+  }
+  return Profile
+}
+```
+
+</tab>
+
+<tab>
+
+```js
+module.exports = (sequelize, DataTypes) => {
+  const Category = sequelize.define("Category", {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  })
+  Category.associate = (models) => {
+    Category.belongsToMany(models.Post, {
+      through: "PostCategories",
+      as: "posts",
+    })
+  }
+  return Category
+}
+```
+
+</tab>
+
+</TabbedContent>
+
+The models have the following relations:
+
+- 1-1: `User` ↔ `Profile`
+- 1-n: `User` ↔ `Post`
+- m-n: `Post` ↔ `Category`
+
+The corresponding tables have been created using a generated Sequelize migration.
+
+In this guide, the route handlers are located in the `src/controllers` directory. The models/entities are located in the `src/models` directory. From there, they are pulled into a central `src/routes.js` file which is used to set up the required routes in `src/index.js`:
```

Comment:
I think you can remove the notion of entities from this guide since Sequelize calls them _models_ by default. I used the term entities in the TypeORM migration guide because in TypeORM models are called "entities".

```suggestion
In this guide, the route handlers are located in the `src/controllers` directory. The models are located in the `src/models` directory. From there, they are pulled into a central `src/routes.js` file which is used to set up the required routes in `src/index.js`:
```